src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const treeHeaders = document.querySelectorAll('.tree > li > :first-child');
+
+// Wrap each header text in a span tag and add a click event
+treeHeaders.forEach((header) => {
+  // Wrap text content in a span tag
+  const span = document.createElement('span');
+
+  span.textContent = header.textContent;
```

Comment:
Consider trimming the text content before assigning it to the span to avoid any leading or trailing whitespace issues.